api/types/image/opts.go

```diff
@@ -103,6 +103,11 @@ type LoadOptions struct {
 	Platforms []ocispec.Platform
 }
 
+type InspectOptions struct {
+	// Manifests returns the image manifests.
+	Manifests bool
+}
+
```

Comment:
It just occurred to me that there's 2 option structs now;

- `backend.ImageInspectOptions` (used by the API server)
- `image.InspectOptions` (used by the client)


~The last one, by design, is not exported; it's made internal through `imageInspectClientOpt.apiOptions` (non-exported type AND non-exported field)~

~Given that those are not exported currently, perhaps we should (for now at least) move  this type to the client, and un-export it?~

:warning: 👉 edit: ignore me; there's `ImageInspectWithOpts`, which allows all options to be passed as a struct